src/assets/i18n/de_DE.json

```diff
@@ -884,9 +884,37 @@
   "product.ready_for_shipment.within24": "Gewöhnlich lieferbar in 24 Stunden",
   "product.remove_all_product_filters.text": "Alle Filter zurücksetzen",
   "product.retailSet.products.heading": "Dieses Retail-Set enthält",
+  "product.review.content.error.required": "Bitte geben Sie einen Text für Ihre Rezension ein.",
+  "product.review.content.label": "Rezension",
+  "product.review.content.max_limit": "Ihre Bewertung: {{0}} Zeichen verbleibend.",
+  "product.review.content.placeholder": "Was gefiel Ihnen und was nicht? Warum haben Sie das Produkt gekauft?",
+  "product.review.name.label": "Name",
+  "product.review.rating.error.required": "Klicken Sie zum Bewerten auf die Sterne.",
+  "product.review.rating.label": "Bewertung",
+  "product.review.rating.tooltip": "{{0, plural, one{# Stern} other{# Sterne}}}",
+  "product.review.status.new.text": "Ihre Rezension wird geprüft. Andere Benutzer können Ihre Rezension erst sehen, wenn sie genehmigt wurde.",
+  "product.review.status.rejected.text": "Ihre Rezension wurde vom Shop-Administrator zurückgewiesen. Sie können die Rezension löschen und eine neue verfassen, wenn Sie möchten.",
+  "product.review.title.error.required": "Bitte geben Sie einen Titel für Ihre Rezension ein.",
+  "product.review.title.label": "Titel",
+  "product.reviews.anonymous_user.text": "Anonymer Benutzer",
+  "product.reviews.create.approval.message": "Danke für Ihre Produktrezension! Ihre Rezension \"{{0}}\" wurde gespeichert. Bevor sie für alle Kunden sichtbar ist, muss sie vom Shop-Administrator geprüft werden.",
+  "product.reviews.create.dialog.cancel.button.label": "Abbrechen",
+  "product.reviews.create.dialog.confirm.button.label": "Speichern",
+  "product.reviews.create.dialog.description.text": "Sagen Sie uns, was Sie über diesen Artikel denken, und teilen Sie Ihre Erfahrung mit anderen. Bitte beschränken Sie sich auf Informationen, die das Produkt betreffen, welches Sie rezensieren.",
+  "product.reviews.create.dialog.heading": "Diesen Artikel bewerten und rezensieren",
```

Comment:
"Artikel" can mean different things in German. Maybe say "Dieses Produkt bewerten und rezensieren" to be more precise?